Clear/EVMState.lean

```diff
@@ -65,7 +65,10 @@ def Account.lookupStorage (act : Account) (k : UInt256) : UInt256 :=
   | _ => 0
 
 def Account.updateStorage (act : Account) (k v : UInt256) : Account :=
-  {act with storage := Finmap.insert k v act.storage}
+  if v == default then
```

Comment:
Why use `default` rather than the explicit value (`0`)? Particularly since we don't mirror this in `Account.lookupStorage`.